### PR TITLE
Update Bischofshofen and Mitterberghütten localities

### PR DESCRIPTION
**Pull request by stepps00**
This PR updates the localities of Bischofshofen and Mitterberghütten, Austria. Specifically:

- Moves the existing Mitterberghütten geometry to an alt geometry
- Updates the Mitterberghütten geometry, moving the record to a Point geometry
- Moves the existing Bischofshofen geometry to an alt geometry
- Updates the Bischofshofen geometry, moving the record to a Polygon geometry (the existing Mitterberghütten geometry)
- Updates label geometries, src properties, etc as needed

This will require PIP work.. once approved, I will PIP the updated records to update descendants.

**Number of records updated**: 2

### CODE REVIEW

File: data/101/844/515/101844515-alt-quattroshapes.geojson [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "id": 101844515,
+  "type": "Feature",
+  "properties": {
+    "src:geom":"whosonfirst",
```
Comment (nvkelso) [inline data/101/844/515/101844515-alt-quattroshapes.geojson:5]: Are these supposed to say `quattroshapes`?

File: data/101/844/515/101844515.geojson [hunk 2]
```diff
@@ -32,15 +30,16 @@
     "qs:pop":0,
     "qs:source":"TCustom EuroGlobalMap + UMZ Urban Polygons + Geonames + GeoPlanet",
     "qs:type":"rural point-near-poly",
-    "src:geom":"quattroshapes",
-    "src:geom_alt":[],
-    "src:lbl_centroid":"mapshaper",
+    "src:geom":"whosonfirst",
+    "src:geom_alt":["quattroshapes"],
+    "wof:geom_alt":["quattroshapes"],
```
Comment (nvkelso) [inline data/101/844/515/101844515.geojson:35]: I'm n sure about this `"wof:geom_alt"` property, is it valid?

File: data/112/578/988/9/1125789889.geojson [hunk 3]
```diff
@@ -211,7 +214,10 @@
     },
     "wof:country":"AT",
     "wof:created":1497290827,
-    "wof:geomhash":"fbe79717e2495b002ac515781addbfbd",
+    "wof:geom_alt":[
```
Comment (nvkelso) [inline data/112/578/988/9/1125789889.geojson:217]: Seems suspicious / duplicative with `src:geom_alt`...

File: data/112/578/988/9/1125789889.geojson [hunk 4]
```diff
@@ -233,10 +239,10 @@
...
...7434],[13.219696,47.406535],[13.221019,47.406496],[13.220963,47.405597],[13.222286,47.405558],[13.222174,47.40376],[13.223497,47.40372],[13.223385,47.401922],[13.224708,47.401883],[13.224429,47.397387],[13.223106,47.397426],[13.222995,47.395628],[13.221672,47.395667],[13.221616,47.394768],[13.218971,47.394847],[13.218915,47.393947],[13.217592,47.393987],[13.21776,47.396684],[13.216437,47.396723],[13.216548,47.398521],[13.215226,47.39856],[13.215337,47.400359],[13.214014,47.400398],[13.21407,47.401297],[13.212747,47.401336],[13.212858,47.403134],[13.21815,47.402978],[13.218317,47.405675],[13.216994,47.405714],[13.21705,47.406613],[13.215727,47.406652],[13.215783,47.407552],[13.213137,47.40763],[13.213304,47.410327],[13.211981,47.410366],[13.212148,47.413063],[13.208178,47.413181],[13.20829,47.414979],[13.209613,47.41494],[13.209668,47.415839],[13.210992,47.4158],[13.211493,47.423892],[13.212816,47.423853],[13.21315,47.429248],[13.211827,47.429287],[13.211938,47.431085],[13.210615,47.431124],[13.210726,47.432922],[13.209402,47.432961],[13.209959,47.441952],[13.211283,47.441913],[13.211394,47.443712],[13.212718,47.443673],[13.212774,47.444572],[13.214098,47.444533]]],"type":"Polygon"}
```
Comment (nvkelso) [inline data/112/578/988/9/1125789889.geojson:247]: New geom seems reasonable

**Review by nvkelso — CHANGES_REQUESTED** [review]
See comments.

The geometry swapping looks good, but some of the properties look funky.

File: data/101/844/515/101844515-alt-quattroshapes.geojson (hunk 1 above)
Comment (stepps00) [reply data/101/844/515/101844515-alt-quattroshapes.geojson]: Updated to `quattroshapes` in the most recent commit.

File: data/101/844/515/101844515.geojson (hunk 2 above)
Comment (stepps00) [reply data/101/844/515/101844515.geojson]: Valid property ([info](https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/geom_alt.json)), but needed to be exportified.. updated in most recent commit.

File: data/112/578/988/9/1125789889.geojson (hunk 3 above)
Comment (stepps00) [reply data/112/578/988/9/1125789889.geojson:217]: Some about the property: https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/geom_alt.json

Issue: https://github.com/whosonfirst-data/whosonfirst-data/issues/1793

This property is basically the `src:geom_alt` prop, but only displays source names of each alt file.

**Comment by stepps00** [comment]
Exportified and confirmed `src:` property values in the most recent commit.. this PR is ready to merge.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**